packages/togetherai/src/index.ts

```diff
@@ -1,5 +1,8 @@
+import { OpenAICompatibleErrorData } from '@ai-sdk/openai-compatible';
+
 export { createTogetherAI, togetherai } from './togetherai-provider';
 export type {
   TogetherAIProvider,
   TogetherAIProviderSettings,
 } from './togetherai-provider';
+export type { OpenAICompatibleErrorData as TogetherAIErrorData };
```

Comment:
I just realized I can probably combine line 1 and this line. Will edit before merge if so.